src/connector_postgres_v2/base_command.py

```diff
@@ -66,8 +65,8 @@ def handler(conn: Any, cursor: Any) -> None:
         return self._execute(sql, conn_str, handler)
 
     def fetchall(self, sql: str, conn_str: str, values: list) -> ConnectorProxyResponseDict:
-        def prep_results(results: dict) -> list:
-            return list(map(list, results))
+        def prep_results(results: list) -> list:
```

Comment:
could you possibly add an example input and output or something along those lines to show what this is doing?

Reply:
np, updated.